src/cli.zig

```diff
@@ -323,6 +362,18 @@ test "parse_size" {
     try expectEqual(@as(u64, 1000 * kib), parse_size("  1000  kb "));
 }
 
+/// Given a limit ilke `10GiB`, return the maximum power-of-two count of `T`s
+/// that can fit in the limit.
+fn parse_size_to_count(comptime T: type, string: []const u8) u32 {
+    const byte_size = parse_size(string);
+    const count = math.floorPowerOfTwo(u64, byte_size / @sizeOf(T));
+    const result = math.cast(u32, count) catch |err| switch (err) {
```

Comment:
I would probably name this `count_u32` — the "_u32" type trailer isn't great, but it keeps the relationship with `count` obvious.